tomlib/tomlib is the native extension (tomlib.so) which is installed and should
be loaded from $GEM_HOME/extensions, not $GEM_HOME/gems along the Ruby files.
Thus `require_relative` shouldn't be used for requiring native extensions.

--- a/lib/tomlib.rb
+++ b/lib/tomlib.rb
@@ -2,9 +2,9 @@
 
 require 'date'
 
-require_relative 'tomlib/dumper'
-require_relative 'tomlib/tomlib'
-require_relative 'tomlib/version'
+require 'tomlib/dumper'
+require 'tomlib/tomlib'
+require 'tomlib/version'
 
 # Main library namespace
 #
